remoteit/desktop
# [DESK-574] Fix uninstaller windows

### Description

uninstaller windows -- Tested windows 7 and 10

  - Delete the folder C:\Program Files\remoteit-bin
  - Delete the folder C:\Users\%username%\AppData\Local\remoteit\ (including all files under it)
  - Delete the folder C:\ProgramData\remoteit\ (including all files under it)

### Checklist




### Test plan



1. Run .exe in windows machine 
2. Uninstall remote.exe
3. You shouldn't see 
  - Delete the folder C:\Program Files\remoteit-bin
  - Delete the folder C:\Users\%username%\AppData\Local\remoteit\ (including all files under it)
  - Delete the folder C:\ProgramData\remoteit\ (including all files under it)

### Ticket
https://remoteit.atlassian.net/browse/DESK-574
